### PR TITLE
[12.0][MIG] sale_procurement_group_by_line

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Review by AaronHForgeFlow — COMMENTED** [review]
Code in models LGTM. Missed to migrate tests.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

File: sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py
Comment (LoisRForgeFlow) [inline sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py:87]: no assert in this test?

Comment (MiquelRForgeFlow) [reply sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py:87]: and what we should assert?

Comment (LoisRForgeFlow) [reply sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py:87]: the expected behavior? :laughing:  If you understand the module you should be able to know what to assert.

Comment (MiquelRForgeFlow) [reply sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py:87]: but that's not constructive 😒

Comment (AaronHForgeFlow) [reply sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py:87]: ¬¬ IMHO just checking two procurement groups are created is ok. Perhaps also the names for the procurment groups...

Comment (MiquelRForgeFlow) [reply sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py:87]: a lot of asserts implemented :)

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

File: sale_procurement_group_by_line/__manifest__.py
Comment (tafaRU) [inline sale_procurement_group_by_line/__manifest__.py]: Shouldn't be `12.0.1.0.0`?

File: sale_procurement_group_by_line/readme/DESCRIPTION.rst
Comment (tafaRU) [inline sale_procurement_group_by_line/readme/DESCRIPTION.rst]: s/Sale/sale

**Review by tafaRU — CHANGES_REQUESTED** [review]
Some nitpicking, the rest LGTM. Thanks!

**Review by tafaRU — APPROVED** [review]

**Comment by tafaRU** [comment]
@mreficent, before to merge I think we need to amend https://github.com/OCA/sale-workflow/pull/978/commits/0d51af94db4d7debba62687fec27e904670267db with an email address registered ([ref.](https://github.com/OCA/sale-workflow/pull/978#issuecomment-547112921)). Could someone of @OCA/core-maintainers please check it?

**Comment by MiquelRForgeFlow** [comment]
@tafaRU The PR cannot remain blocked just for that reason. It happens a lot. This commit you refer is very old and it's expected that people may change their email with time. If you have the new email, I can do a fast amend. If not, then it remains as it is.

**Comment by tafaRU** [comment]
> If you have the new email, I can do a fast amend.

I don't have it, this is the reason why I asked!

> The PR cannot remain blocked just for that reason. It happens a lot.

Ok, I agree, I proceed with the merge then.

**Pull request merged**
